contrib/openapi/spec.yml

```diff
@@ -1,453 +1,282 @@
 swagger: '2.0'
+schemes:
+  - http
+host: api.xsnippet.org
 info:
   title: XSnippet API
-  description: REST API of XSnippet, a simple web-service for sharing code snippets
-  version: "4.0.0"
+  version: '1.0'
+  contact:
+    email: dev@xsnippet.org
+  description: |
+    The XSnippet API is organized around [REST]. Our API has predictable,
+    resource-oriented URLs, and uses HTTP response codes to indicate API
+    errors. We use built-in HTTP features, like HTTP authentication and
```

Comment:
According to [RFC-7235], HTTP provides a general framework for access control and authentication, via an extensible set of challenge-response authentication schemes, and `bearer` scheme is one of them (see [IANA Auth Schemes]). So I think it's correct to say that we support HTTP authentication. Otherwise, please provide better wording or suggest to remove it. :)

[RFC-7235]: https://tools.ietf.org/html/rfc7235
[IANA Auth Schemes]: https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml